### ISSUE TITLE
Quick QoL Request

### ISSUE THREAD

**Issue by kd8lvt** [Azanor/thaumcraft-beta#301]
I've done this a couple times now, and it causes a lot of flux. Could you disable the "right click an item on the crucible to put it in" for the Thaumonomicon and Thaumometer? Or maybe a config option for a list of items to disable that for? Would be very helpful when doing the Alchemy research, as it involves a lot of back and forth with the Thaumonomicon and the Crucible.

**Comment by apmax** [comment]
The right click function is incredibly useful for quartz bit -> vis shard crafting.  The final stage of alchemy crafting in general I guess, but almost nothing else.
I think some sort of blacklist for the function would probably be a good idea.  Or optionally, if you cannot craft a recipe with the item in your hand and the essentia in the crucible, right clicking does nothing.

**Comment by Ulysses2** [comment]
If right clicking was only for crafting things with the crucible and not melting things down, it’d be really convenient. But on the other hand, the crucible is supposed to have an element of risk to it. You wouldn’t be in danger of wasting ingredients if you got the recipe wrong.

**Comment by twothe** [comment]
It is ok that the crucible has a certain risk to it, but toppling over my own feet and accidentally throwing my fully enchanted diamond sword into it should not be one of those 'risks'

**Comment by taggedjc** [comment]
I didn't even know you could right click, because I was pretty sure I had tried that and it didn't do anything.

**Comment by kd8lvt** [comment]
@taggedjc  It's mainly used, as stated above, for the quartz shard to vis shard recipe, and it is very useful for that.

Some sort of whitelist/blacklist option would be super useful for pack makers/players. Sure, it introduces _some_ risk, but I don't think it's the right kind of risk. Maybe a "twitch"/"tic" warp event localized near cauldrons where you drop a random item from your hotbar into the cauldron could be interesting, (Or even not localized, you just drop a random/held item) but that could get **very** annoying **very** quickly.

**Comment by Ulysses2** [comment]
No, that would mean you'd be unfairly penalized (sometimes extremely so) for something you had no control over. TBH, since quartz shards don't have aspects and therefore won't be dissolved in the crucible, you don't ever need to right-click on the crucible, even if it does save some hassle.

**Comment by twothe** [comment]
Yeah I just always throw in a full stack of quartz shards and collect the remaining ones later. The only situation where I see that this would be useful is if you want just specific aspects out of the Crucible so you carefully put them in one by one. But that rare situations imo does not justify loosing valuable gear by accident.

**Comment by kd8lvt** [comment]
That's true. I did say an alternative, which was to have it just drop at any random time anywhere, which still introduces risk (to warp, anyways). 
The risk introduced by this mechanic is an annoyance more than it is true risk. A quote from one of my friends while we were playing: "Damnit, I melted my book again!" Its not something someone can break the habit of quickly. Most people are used to the crucible being a non-interactable block, so when it suddenly has the capability to absorb any item in your hand, it takes a while to get used to (and accidents still happen after that).
I still think having a whitelist/blacklist option for crucible right clicking would be a good resolution for this issue. Leave it up to the players, rather than a hard-coded list of items.
